create-directories.bash

```diff
@@ -48,12 +48,10 @@ fi
 realSource="$(realpath -m "$sourceBase$target")"
 if [[ ! -d "$realSource" ]]; then
     printf "Warning: Source directory '%s' does not exist; it will be created for you with the following permissions: owner: '%s:%s', mode: '%s'.\n" "$realSource" "$user" "$group" "$mode"
-    mkdir --mode="$mode" "$realSource"
-    chown "$user:$group" "$realSource"
+    mkdir "$realSource"
 fi
 
 [[ -d "$target" ]] || mkdir "$target"
 
-# synchronize perms between source and target
-chown --reference="$realSource" "$target"
-chmod --reference="$realSource" "$target"
+chown "$user:$group" "$realSource" "$target"
```

Comment:
See commit message for reasoning.

Reply:
I'm a bit skeptical of this, since it would apply mode changes to already existing directories, whether a mode is supplied for the directory or not. I'm not really sure what's not working for you, though. Are the initial directories created with the incorrect permissions?

Reply:
Also, I guess this isn't critical for the main purpose of this PR, so maybe it could be moved to its own?